Modules/CalcTools.lua

```diff
@@ -131,6 +131,8 @@ function calcLib.getGemStatRequirement(level, isSupport, multi)
         if multi == 100 then
             a = 2.1
             b = 7.75
+		elseif multi == 75 then
```

Comment:
The indentation here is a bit off. Possibly due to mixing tabs and spaces?

Reply:
I used tabs but they are heavily mixed with spaces in this area. 
![1](https://user-images.githubusercontent.com/18018499/100073558-f13b2500-2e3d-11eb-95aa-534095bea006.png)

Reply:
Please feel free to change the formatting in quesion to tabs instead.